### DISCUSSION TITLE
Change language

### DISCUSSION THREAD

**Discussion by bernique** [nsh07/WikiReader#7] [Ideas]
Very nice application, efficient and lean. Would that be possible to add an option to change the language of Wikipedia ?
Thank you

**Comment by nsh07** [comment]
Sure, I was planning to add an option to change languages.

Thanks for using my app!

**Comment by nsh07** [comment]
Added an issue for this feature request (#9)

**Comment by nsh07** [comment]
Implemented in v1.8.0 🎉
